src/Composer/InstallFilesPlugin.php

```diff
@@ -80,10 +84,23 @@ public function postPackageInstall(PackageEvent $event): void
         }
 
         $name = $operation->getPackage()->getName();
+
+        $this->io->write(sprintf('<fg=yellow>Création des fichiers de configuration :</>'));
+
         foreach (self::FILES as $file) {
+            if (true === $filesystem->exists($file)) {
+                $answer = $this->io->ask(sprintf('Le fichier de configuration %s existe déjà. Voulez-vous le conserver ? [<fg=yellow>o,n</>] ', $file));
```

Comment:
Je voulais te signaler qu'il existe une classe Symfony, appelée [ConfirmationQuestion](https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/Console/Question/ConfirmationQuestion.php), qui permet de valider les retours de l'utilisateur par "oui" ou "non" lors de l'exécution de la console. N'hésite pas à jeter un coup d'œil à la [documentation](https://symfony.com/doc/current/components/console/helpers/questionhelper.html#asking-the-user-for-confirmation) et à considérer cette option pour améliorer l'expérience utilisateur de ta console. Si tu as besoin d'aide pour l'implémenter, je suis là pour t'aider.

Reply:
_ConfirmationQuestion_ nécessite de passer par une commande pour bénéficier du _InputInterface_ et du _OutputInterface_

Reply:
_ConfirmationQuestion_ ne peut être utilisé que dans le contexte d'une commande a priori. J'ai utilisé _askConfirmation_ qui est l’équivalent sur un _IOInterface_